OpenSim/Analyses/JointReaction.cpp

```diff
@@ -670,8 +628,7 @@ record(const SimTK::State& s)
     /* Write the reaction data to storage*/
     _storeReactionLoads.append(s.getTime(),_Loads.getSize(),&_Loads[0]);
 
-
-    return(0);
+    return 0;
```

Comment:
This section is a good subject for a future revision, outside of this PR.  When I originally put this logic together, it was necessary because simbody's interface only reported all mobilizer reaction forces and moments at once.  Now, the interface is much better, allowing you to ask a mobilizer for just it's reaction force, and in the reference frame you chose.  Most of this logic isn't necessary anymore.

A future update could:
1. Define a serializable JointReactionReference class, which is the name of the joint, receiving frame, and express_in_frame for specifying exactly the reaction you want to know.
2. Simply define a list property of JointReactionReferences
3. Replace all the logic above with a loop through the list, ask each mobilizer for it's reaction, and rotate the vectors to the reference frame of choice.

Like I said.  For down the road, unless all the analyses fade of below the horizon eventually.